### PR TITLE
Ks json schema

### PR DESCRIPTION
**Pull request by kms617**
Json schema implemented for events and nearests spec tests.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -14,7 +14,7 @@
 ActiveRecord::Schema.define(version: 20150113183017) do
 
   # These are extensions that must be enabled in order to support this database
-  enable_extension "plpgsql"
+  # enable_extension "plpgsql"
```
Comment (edwardloveall) [inline db/schema.rb:17]: Did you comment this out manually or did something else comment it

**Comment by edwardloveall** [comment]
Just concerned about the `enable_extension` bit. Let me know why that's commented. Otherwise looks good to merge

File: db/schema.rb (hunk 1 above)
Comment (kms617) [inline db/schema.rb:17]: something else commented this out

File: spec/models/event_spec.rb [hunk 2]
```diff
@@ -7,6 +7,7 @@
 
   describe 'validations' do
     it { should validate_presence_of(:lat) }
+    it { should validate_presence_of(:lat) }
```
Comment (dthyresson) [inline spec/models/event_spec.rb:10]: Did this line duplicate?

Comment (dthyresson) [inline spec/models/event_spec.rb:10]: Ah. Think you might mean `address`?

**Pull request closed without merging**